src/extension/background-script/actions/lnurl/__tests__/auth.test.ts

```diff
@@ -1,39 +1,93 @@
 import { settingsFixture as mockSettings } from "~/../tests/fixtures/settings";
+import Mnemonic from "~/extension/background-script/mnemonic";
 import state from "~/extension/background-script/state";
+import { btcFixture } from "~/fixtures/btc";
 import type { LNURLDetails } from "~/types";
 
-import { authFunction } from "../auth";
+import { authFunction, getPathSuffix } from "../auth";
 
+let fetchedUrl: string;
 jest.mock("~/extension/background-script/state");
+jest.mock("axios", () => ({
+  get: (requestUrl: string) => {
+    fetchedUrl = requestUrl;
+    return { data: { status: "OK" } };
+  },
+}));
 
-const mockState = {
-  settings: mockSettings,
-  getConnector: () => ({
-    signMessage: () =>
-      Promise.resolve({
-        data: {
-          signature:
-            "rnu5pnhanjs3bfxz33fuyf9ywzrmkm1ns6jxdraxff1irq3hpxcbkce6zk34ee9bh7bamgd891tfy4gq1y119w53qg1ap5zodwi4u51n",
-        },
+const passwordMock = jest.fn;
+
+describe("auth with mnemonic", () => {
+  test("getPathSuffix matches test vector", () => {
+    expect(
+      getPathSuffix(
+        "site.com",
```

Comment:
Nit: Use example.com instead of some random domains.

Reply:
@reneaaron this comes from the test vector. I cannot change it otherwise the derivation path would change